Makefile

```
@@ -1,50 +1,56 @@
-.PHONY: all build clean fmt fmt-check init linter pre-commit test
+.PHONY: all build clean fmt fmt-check init lint pre-commit test full-test
 
 all: init build test
 
 build:
-	@echo ──────────── Build release ────────────────────
-	@cargo +nightly build --release
-	@ls -l ./target/wasm32-unknown-unknown/release/*.wasm
-
-clean:
-	@echo ──────────── Clean ────────────────────────────
-	@rm -rvf target
+	@echo ⚙️ Building a release...
+	@cargo +nightly b -r
+	@ls -l target/wasm32-unknown-unknown/release/*.wasm
 
 fmt:
-	@echo ──────────── Format ───────────────────────────
+	@echo ⚙️ Formatting...
 	@cargo fmt --all
 
 fmt-check:
-	@echo ──────────── Check format ─────────────────────
-	@cargo fmt --all -- --check
+	@echo ⚙️ Checking a format...
+	@cargo fmt --all --check
 
 init:
-	@echo ──────────── Install toolchains ───────────────
+	@echo ⚙️ Installing a toolchain \& a target...
 	@rustup toolchain add nightly
 	@rustup target add wasm32-unknown-unknown --toolchain nightly
 
-linter:
-	@echo ──────────── Run linter ───────────────────────
-	@cargo +nightly clippy --all-targets -- --no-deps -D warnings -A "clippy::missing_safety_doc" -D warnings -A "clippy::too_many_arguments"
+lint:
+	@echo ⚙️ Running the linter...
+	@cargo +nightly clippy --all-targets -- -D warnings
```

```suggestion
	@cargo +nightly clippy --workspace --all-targets -- -D warnings
```
To run the linter on the `state` crate too.